src/libpmemobj/lane.c

```diff
@@ -63,7 +63,7 @@ struct section_operations *Section_ops[MAX_LANE_SECTION];
 /*
  * lane_info_destroy -- destroy lane info hash table
  */
-inline void
+void
```

Comment:
Yes, also the other function below should be `static` instead of `static inline`.
But that is not related to ICC, so whatever.